screens/Marketplace.js

```diff
@@ -14,12 +14,71 @@ class MarketplaceBase extends Component {
     active: 'Marketplace', 
     items: [],
     type: 'buying', // the type of items being displayed, default buying
-  }
 
-  componentDidMount() {
-    this.setState({ items: this.props.items });
+    // clair
```

Comment:
Is line 19 repetitive with line 16? If field in line 16 is not used, remove it

Reply:
they r both being used. let's leave them like that now. thank u Cheng 😁